pkg/restore/restore.go

```diff
@@ -96,6 +97,13 @@ type Restorer interface {
 		snapshotLocationLister listers.VolumeSnapshotLocationLister,
 		volumeSnapshotterGetter VolumeSnapshotterGetter,
 	) (Result, Result)
+	RestoreWithResolvers(
```

Comment:
rename the parm resolver to make it more specific to restoreItemAction?

Reply:
Sure